### ISSUE TITLE
cryptocoins-icons.woff2 and cryptocoins-icons.svg not found

### ISSUE THREAD

**Issue by Burmudar** [allienworks/cryptocoins#63]
Hi,

When I include `cryptocoins-icons.css` it can't find the following files:

- `cryptocoins-icons.woff2`
- `cryptocoins-icons.svg`

Contents of `node_modules/cryptocoins/webfont`:
```bash
./webfont/cryptocoins.css
./webfont/cryptocoins-colors.css
./webfont/cryptocoins-icons.woff
./webfont/cryptocoins-icons.ttf
./webfont/cryptocoins-icons.eot
```

**Comment by VLEFF** [comment]
Hi,
Same for me, I ended up removing those two lines from the css.
It's an ugly workaround I guess, but it works ^^

**Comment by yurik94** [comment]
You could generate the woff2 here:

https://everythingfonts.com/ttf-to-woff2

**Comment by allienworks** [comment]
Thanks for notice and sorry for late reply. This is a mistake made by automatic webfont generator (my bad), which will be fixed in next (2.6) release.

In the end I went with TTF, WOFF and WOFF2 only, as those cover [fair share of browser usage](https://www.w3schools.com/css/css3_fonts.asp).
